funcx_endpoint/funcx_endpoint/endpoint/endpoint.py

```diff
@@ -298,7 +299,7 @@ def start_endpoint(
                                        )
     except Exception as e:
         logger.critical("Caught exception while trying to setup endpoint context dirs")
-        logger.critical("Exception : ", e)
+        logger.critical("Exception: ", e)
```

Comment:
This could be replaced by a single `logger.exception` line which reports the exception info automatically,

Reply:
Updated.